src/replication.cc

```diff
@@ -65,6 +65,7 @@ void FeedSlaveThread::checkLivenessIfNeed() {
 }
 
 void FeedSlaveThread::loop() {
+  bool send_by_batch = false;
```

Comment:
how about renaming the `send_by_batch ` to `is_first_repl_batch` with a comment on why we need it

```
// is_first_repl_batch was used to fix that replication may be stuck in a dead loop 
// when some seqs might be lost in the middle of the WAL log, so forced to replicate
// first batch here to work around this issue instead of waiting for enough batch size. 
bool is_first_repl_batch = false;
```